.travis/Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN cd /build && \
 	git clone -b $(curl -s -L http://grpc.io/release) https://github.com/grpc/grpc && \
 	cd /build/grpc && \
 	git submodule update --init && \
-	make -j && \
+	make -j4 && \
```

Comment:
I think if a VM has 4GB of memory and 2GB out of 4 is dedicated to hugepages, the actual amount of available memory could be very small. Not sure how big a typical g++ process would take, but excessive swapping might happen even with 2 processes.